ast/compile.go

```diff
@@ -883,6 +883,110 @@ func compileSchema(goSchema interface{}) (*gojsonschema.Schema, error) {
 	return schemasCompiled, nil
 }
 
+func mergeSchemas(schemas []*gojsonschema.SubSchema) (*gojsonschema.SubSchema, error) {
+	if len(schemas) == 0 {
+		return nil, nil
+	}
+	var result = schemas[0]
+
+	for i := 0; i < len(schemas); i++ {
+		if len(schemas[i].PropertiesChildren) > 0 {
+			if !schemas[i].Types.Contains("object") {
+				schemas[i].Types.Add("object")
```

Comment:
See Go Lint test results here: https://github.com/aavarghese/opa/pull/1/checks?check_run_id=3272579440

Its asking you to make the following code style changes:

ast/compile.go:895:25: Error return value of `.Add` is not checked (errcheck)
				schemas[i].Types.Add("object")
				                    ^
ast/compile.go:899:25: Error return value of `.Add` is not checked (errcheck)
				schemas[i].Types.Add("array")